scripts/jenkins-yoctobuild-build.sh

```diff
@@ -2,19 +2,24 @@
 
 set -e -x -E
 
-$WORKSPACE/mender-qa/scripts/wait-until-build-host-ready.sh
+if [ -n "$JENKINS_URL" ]; then
+    $WORKSPACE/mender-qa/scripts/wait-until-build-host-ready.sh
+fi
 
 echo $WORKSPACE
 
-sudo apt-get -qy --force-yes install docker-ce || apt-get -qy --force-yes install docker-ce
-# make sure Docker uses the block storage to store docker containers.
-sudo bash -c 'cat << EOF > /etc/docker/daemon.json
-{
-"data-root": "/var/lib/docker/",
-"storage-driver": "overlay2"
-}
-EOF'
-sudo service docker restart
+# Jenkins builds will install docker here. Gitlab runners have it already in the image
+if ! hash docker 2>/dev/null; then
```

Comment:
One thing to discuss.

To be consistent, we could change this if condition to `if [ -n "$JENKINS_URL" ]; then` (as the rest). It is a bit more obscure in that why do we do it for Jenkins, but it will easier signal which parts to remove when we finally fork and abandon Jenkins support.

Reply:
Yes, good point, I think I would do that instead, since we expect to remove this when Jenkins is gone.

Reply:
Done :+1: